x-pack/test/security_solution_cypress/cypress/README.md

```diff
@@ -42,7 +42,10 @@ Please, before opening a PR with the new test, please make sure that the test fa
 
 Note that we use tags in order to select which tests we want to execute:
 
-- `@serverless` includes a test in the Serverless test suite. You need to explicitly add this tag to any test you want to run against a Serverless environment.
+*DISCLAIMER: We are adding the @serverlessQA tag temporaly until we check the behaviour of our tests in the second quality gate.*
```

Comment:
What's the longer-term plan? We would run all tests tagged as `@serverless` in QA?

Reply:
That is the initial idea :) 

That being said, we need also to check the feasibility and if truly makes sense. Also if we need to add tests we cannot currently have on the first quality gate, for instance, creating a project from scratch.

Reply:
@MadameSheema Thank you, that makes sense 👍